### PR TITLE
update directory components

### PR DESCRIPTION
**Pull request by Alizaaaja4**
- add directory components admins

### CODE REVIEW

File: components/components_admins/arcodning.tsx
Comment (Asady04) [inline components/components_admins/arcodning.tsx:1]: good

**Review by Asady04 — APPROVED** [review]
god joob, thankyou

**Comment by Alizaaaja4** [comment]
> god joob, thankyou

bisa aku merged ?

**Comment by Asady04** [comment]
> > god joob, thankyou
> 
> bisa aku merged ?

terserah

**Pull request merged**